src/test/java/com/library/utils/IsSameLikeBook.java

```diff
@@ -0,0 +1,22 @@
+package com.library.utils;
+
+import com.library.model.entity.Book;
+import org.mockito.ArgumentMatcher;
+
+public final class IsSameLikeBook implements ArgumentMatcher<Book> {
```

Comment:
Please create this service more abstract. With parametrized type. Like it was done in Validate service.

Reply:
ok